### PR TITLE
test/old: Removed commented code

### PR DESCRIPTION
**Pull request by joscollin**
There were lot of commented code inside src/test/old/testcrush.cc. This was found while doing
the PR#15364. So Removed them.

Signed-off-by: Jos Collin <jcollin@redhat.com>

### CODE REVIEW

File: src/test/old/testcrush.cc [hunk 1]
```diff
@@ -161,8 +138,6 @@ int main()
     root = make_hierarchy(c, wid, ndisks, nbuckets);
   }
   
-
-
   // rule
```
Comment (amitkumar50) [inline src/test/old/testcrush.cc]: I believe this can also be removed...

**Comment by joscollin** [comment]
@amitkumar50 Those blank lines are unnecessary. Only one blank line is expected. And this is just a cleanup PR -> No need to create another PR just for removing those blank lines.

File: src/test/old/testcrush.cc [hunk 2]
```diff
@@ -44,7 +28,6 @@ Bucket *make_bucket(Crush& c, vector<int>& wid, int h, int& ndisks, int& nbucket
     UniformBucket *b = new UniformBucket(nbuckets--, 1, 0, 10, disks);
     b->make_primes(hash);  
     c.add_bucket(b);
-    //cout << h << " uniformbucket with " << wid[h] << " disks" << endl;
     return b;
   } else {
     // mixed
```
Comment (amitkumar50) [inline src/test/old/testcrush.cc]: mixed can also be removed

Comment (joscollin) [inline src/test/old/testcrush.cc]: @amitkumar50 

That is an informative comment in the `if {} else {}` that indicates UniformBucket or MixedBucket is to be made. It improves the readability of the code too, when someone reads it for the first time.

**Pull request merged**